tests/scale/test_upgrade_scale.py

```
@@ -0,0 +1,453 @@
+        resource_class=VirtualMachine,
+        idle_requests_value=VIRT_HANDLER_API_IDLE_STATE * len(vms_for_upgrade_test),
+    )
+
+
+@pytest.fixture()
+def idle_monitored_api_requests(monitor_api_requests_object):
+    monitor_api_requests_object.wait_for_idle()
+
+
+@pytest.fixture(scope="module")
+def running_vms_with_load(running_vms_for_upgrade_test, stopped_nginx_vm):
```

Ruff (0.14.8)

Unused function argument: `stopped_nginx_vm`

(ARG001)